bindings/core/src/method/client.rs

```diff
@@ -153,34 +139,33 @@ pub enum ClientMethod {
     /// Expected response: [`LedgerNanoStatus`](crate::Response::LedgerNanoStatus)
     #[cfg(feature = "ledger_nano")]
     #[cfg_attr(docsrs, doc(cfg(feature = "ledger_nano")))]
+    #[serde(rename_all = "camelCase")]
     GetLedgerNanoStatus {
         /// To use a Ledger Speculos simulator, pass `true` to `is_simulator`; `false` otherwise.
-        #[serde(rename = "isSimulator")]
         is_simulator: bool,
     },
     /// Prepare a transaction for signing
+    #[serde(rename_all = "camelCase")]
     PrepareTransaction {
         /// Secret manager
         #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
-        #[serde(rename = "secretManager")]
         secret_manager: Option<SecretManagerDto>,
         /// Options
         options: Option<BuildBlockOptions>,
     },
     /// Sign a transaction
+    #[serde(rename_all = "camelCase")]
     SignTransaction {
         /// Secret manager
-        #[serde(rename = "secretManager")]
         #[derivative(Debug(format_with = "OmittedDebug::omitted_fmt"))]
         secret_manager: SecretManagerDto,
         /// Prepared transaction data
-        #[serde(rename = "preparedTransactionData")]
         prepared_transaction_data: PreparedTransactionDataDto,
     },
     /// Build a block containing the specified payload and post it to the network.
+    #[serde(rename_all = "camelCase")]
     PostBlockPayload {
         /// The payload to send
-        #[serde(rename = "payload")]
         payload_dto: PayloadDto,
```

Comment:
Renaming to payload was good, otherwise it's breaking?

Reply:
How annoying. Do we prefer unifying the naming or avoiding a breaking change?

Reply:
Bindings build this and for them it's not a Dto, but just the normal payload, so I would keep it non breaking